### PR TITLE
added issue forms

### PR DESCRIPTION
**Pull request by sdushantha**
Added issues forms so that it is easier for users to write issues

### CODE REVIEW

**Review by ppfeister — CHANGES_REQUESTED** [review]
For F+/F- reports, it may be beneficial to have an optional field for username(s) effected

If a site is susceptible to F+/F- with a specific pattern only (i.e. dots), that may speed up testing & patching

File: .github/ISSUE_TEMPLATE/site-request.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Site Support 
+description: Request a site Sherlock should support
+labels: ["site request"]
```
Comment (ppfeister) [inline .github/ISSUE_TEMPLATE/site-request.yml:3]:
```suggestion
labels: ["site support request"]
```

Match current label

**Review by ppfeister — CHANGES_REQUESTED** [review]
Minor correction to label

**Comment by sdushantha** [comment]
> For F+/F- reports, it may be beneficial to have an optional field for username(s) effected
> 
> If a site is susceptible to F+/F- with a specific pattern only (i.e. dots), that may speed up testing & patching

I was thinking that such information could be added to the Additional Information textbox

**Comment by ppfeister** [comment]
> > For F+/F- reports, it may be beneficial to have an optional field for username(s) effected
> > If a site is susceptible to F+/F- with a specific pattern only (i.e. dots), that may speed up testing & patching
> 
> I was thinking that such information could be added to the Additional Information textbox

Having dealt with the general public while trying to diagnose infra issues and such, I'm of the belief that very explicit directions are sometimes needed, else you get reports like #2126 "some of these don't work, but I'm not going to tell you which, btw this product sucks".

Although those types of reports are fairly rare here, so it's not a hard requirement. I think most contributors have been pretty good about their bug reports and it can probably be left out for now.

**Review by ppfeister — APPROVED** [review]
GTG

**Pull request merged**